plugins/CoreHome/vue/src/ContentBlock/ContentBlock.vue

```diff
@@ -78,7 +79,7 @@ export default defineComponent({
       }
     }, 0);
 
-    if (this.actualFeature && (this.actualFeature === true || this.actualFeature === 'true')) {
+    if (this.actualFeature && this.actualFeature === 'true') {
```

Comment:
Yes, actualFeature is initialized to this.feature which is typed to be a String, so the value will never be a boolean `true`. I don't think it was possible in angularjs either since the binding value was `@`, not `=` or `<`. Typescript complained about this being unnecessary if I recall correctly.